frame/crowdloan-rewards/src/lib.rs

```diff
@@ -372,6 +372,17 @@ pub mod pallet {
 		}
 	}
 
+	/// Returns the amount available to claim for the specified account.
+	pub fn amount_available_to_claim_for<T: Config>(
+		account_id: <T as frame_system::Config>::AccountId,
+	) -> Result<T::Balance, DispatchError> {
+		let association = Associations::<T>::get(account_id).ok_or(Error::<T>::NotAssociated)?;
+		let reward = Rewards::<T>::get(association).ok_or(Error::<T>::NothingToClaim)?;
+		let should_have_claimed = should_have_claimed::<T>(&reward)?;
```

Comment:
may could have claimed? because she is not should do it, but could.

Reply:
I just named the variable after the function, I do agree that it could do with a better name (function and variable)

Reply:
[😞](https://emojikeyboard.org/copy/Disappointed_Face_Emoji_%F0%9F%98%9E?utm_source=extlink)

Reply:
The term initially refer to the fact that the use was currently claiming (do_claim). In the RPC the user is not claiming but rather checking the current available claimable amount so you can change it. I don't have a strong opinion it's up to you

Reply:
As a contributor, you 'should' claim this amount, as this reward is the only consequence of being a contributor

Reply:
I think the wording is accurate

Reply:
I think if it is to be changed, it's outside the scope of this PR and can be reviewed later

Reply:
Yeah, I can open a new PR regarding that. This PR was just for a hotfix for the RPC